### DISCUSSION TITLE
[Request Idea] Place Holders for inventory? :)

### DISCUSSION THREAD

**Discussion by EmetK** [runelite/runelite#18208] [Ideas]
Just wondering if it'd be possible to make a plugin that allows place holders for the inventory but for items you want to select.

**Comment by cdfisher** [comment]
Plugins that would affect the positions that items are placed in one's inventory wouldn't be allowed, on the basis that there's no way to implement such a thing that would be compliant with Jagex's third-party client guidelines.
